main.js

```diff
@@ -3,8 +3,9 @@
 
 'use strict';
 
-// The real profile variable declaration in config.ts is folded away by closure. Until we
+// The real profile and release variable declaration in config.ts is folded away by closure. Until we
 // make closure process this file also, make sure that |profile| is defined in this file.
+var release;
 var profile;
 var jvm = new JVM();
```

Comment:
Nit: I would also put a space after the dummy declarations to make it clear that the *jvm* declaration is unrelated to the comment.